### PR TITLE
match structure of json and core json

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5197, /bcgov/entity#5198

*Description of changes:*
 - fixed json to match schema

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/services/business_details_version.py [hunk 1]
```diff
@@ -324,9 +329,11 @@ def party_role_revision_json(transaction_id, party_role_revision) -> dict:
         party_revision = VersionedBusinessDetailsService.get_party_revision(transaction_id, party_role_revision)
         party = {
             **VersionedBusinessDetailsService.party_revision_json(transaction_id, party_revision),
-            'appointmentDate': datetime.date(party_role_revision.appointment_date).isoformat(),
             'cessationDate': cessation_date,
```
Comment (lekshmimallika-aot) [inline legal-api/src/legal_api/services/business_details_version.py:332]: cessation date is also for a role

**Comment by lekshmimallika-aot** [comment]
Is COD filing and AR working as expected with these changes. Please retrieve a completed cod filing and compare whether the returned json is same as the filing json

File: legal-api/src/legal_api/services/business_details_version.py (hunk 1 above)
Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/services/business_details_version.py:332]: so it should be part of roles object?

Comment (lekshmimallika-aot) [reply legal-api/src/legal_api/services/business_details_version.py:333]: for IA, correction etc it is part of roles object. For COD, AR with COD roles doesnt exist and all these attributes (role, appointment date and cessation date ) are in the party object

**Review by lekshmimallika-aot — APPROVED** [review]
Please have a follow up PR that removes the attributes not applicable for IA and COD

**Pull request merged**